released/packages/coq-num-analysis/coq-num-analysis.1.0.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Numerical Analysis in Coq"
+name: "coq-num-analysis"
+version: "1.0.0"
+maintainer: "MILC project <milc@inria.fr>"
+authors: [ "Sylvie Boldo" "François Clément" "Vincent Martin" "Micaela Mayero" "Florian Faissole" "Houda Mouhcine" "Louise Leclerc" "Stéphane Aubry" ]
+homepage: "https://lipn.univ-paris13.fr/coq-num-analysis/"
+dev-repo: "git+https://lipn.univ-paris13.fr/coq-num-analysis/"
+bug-reports: "https://lipn.univ-paris13.fr/coq-num-analysis/issues"
+license: "LGPL-3.0-or-later"
+tags: [ "keyword:real analysis" "keyword:Lax Milgram" "keyword:Lebesgue" "keyword:Tonelli" "keyword:Bochner"]
+build: [make]
+install: [make "install"]
+remove: [make "uninstall"]
```

Comment:
If this is a regular Coq library, it's best to let opam take care of tracking files for installation and removal.
```suggestion
```